docker-entrypoint.sh

```diff
@@ -20,6 +20,10 @@ define('PSM_DB_PREFIX', '${MYSQL_DATABASE_PREFIX}');
 ?>
 EOF
 
+	if [ ${PSM_BASE_URL} != false ]; then
+	sed -i "s|?>|define('PSM_BASE_URL', '$PSM_BASE_URL');\n?>|1" ${APACHE_DOCUMENT_ROOT}/config.php
```

Comment:
What is the ``1`` for? I've never seen that before.

Reply:
it means replace first ocurrence only
it was in the sed command I copied from so I just left it.
`g` would also work obviously